### PR TITLE
[AIRFLOW-410] Add 2 Q/A to the FAQ in the docs

### PR DESCRIPTION
**Pull request by mistercrunch**
Also changed the markup of Questions as sections to be directly linkable. 

I made sure the `rst` rendered nicely here:
screen shot 2016-08-10 at 9 53 27 am

@r39132  @aoen @plypaul

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -103,3 +107,36 @@ Also important to note is that the tasks ``start_date``, in the context of a
...
+simple dictionary.
+
+.. code:: python
+
+    for i in range(10):
+        dag_id = 'foo_{}'.format(i)
+        globals()[dag_id] = DAG(dag_id)
+        # or better, call a function that returns a DAG object!
+
+What's are all the ``airflow run`` commands in my process list?
```
Comment (aoen) [inline docs/faq.rst]: Nit: Remove 
's

Comment (aoen) [inline docs/faq.rst]: I wonder if at some point we should split up the documentation into internals/admin/operations and user-facing (e.g. how to use the UI/see why a task is stuck vs looking at the process list).

**Comment by aoen** [comment]
Fix and LGTM.

File: docs/faq.rst [hunk 2]
```diff
@@ -103,3 +107,36 @@ Also important to note is that the tasks ``start_date``, in the context of a
...
+---------------------------------------------------------------
+
+There's many layers of ``airflow run`` commands, meaning it can call itself.
+
+- Basic ``airflow run``: fires up an executor, and tell it to run an
+  ``airlfow run --local`` command. if using Celery, this means it puts a
+  command in the queue for it to run remote, on the worker. If using
+  LocalExecutor, that translates into running it in a subprocess pool.
+- Local ``airflow run --local``: starts an ``airflow run --raw``
+  command (described bellow) as a subprocess and is in charge of
```
Comment (ldct) [inline docs/faq.rst:138]: spelling: below

File: docs/faq.rst [hunk 3]
```diff
@@ -103,3 +107,36 @@ Also important to note is that the tasks ``start_date``, in the context of a
...
+  emitting heartbeats, listening for external kill signals
+  and insures some cleanup takes place if the subprocess fails
```
Comment (ldct) [inline docs/faq.rst]: nit: ensures

File: docs/faq.rst [hunk 4]
```diff
@@ -103,3 +107,36 @@ Also important to note is that the tasks ``start_date``, in the context of a
...
+        globals()[dag_id] = DAG(dag_id)
+        # or better, call a function that returns a DAG object!
+
+What's are all the ``airflow run`` commands in my process list?
+---------------------------------------------------------------
+
+There's many layers of ``airflow run`` commands, meaning it can call itself.
+
+- Basic ``airflow run``: fires up an executor, and tell it to run an
+  ``airlfow run --local`` command. if using Celery, this means it puts a
```
Comment (ldct) [inline docs/faq.rst]: spelling: airflow

File: docs/faq.rst [hunk 5]
```diff
@@ -103,3 +107,36 @@ Also important to note is that the tasks ``start_date``, in the context of a
...
+
+    for i in range(10):
+        dag_id = 'foo_{}'.format(i)
+        globals()[dag_id] = DAG(dag_id)
+        # or better, call a function that returns a DAG object!
+
+What's are all the ``airflow run`` commands in my process list?
+---------------------------------------------------------------
+
+There's many layers of ``airflow run`` commands, meaning it can call itself.
```
Comment (ldct) [inline docs/faq.rst]: nit: there are

File: docs/faq.rst (hunk 1 above)
Comment (mistercrunch) [inline docs/faq.rst]: I agree about breaking it down, though often with Airflow it's the same people authoring/administrating.

**Comment by aoen** [comment]
LGTM

**Pull request closed without merging**